src/components/AccountDetails/index.tsx

```diff
@@ -311,7 +311,7 @@ export default function AccountDetails({
               <AccountGroupingRow>
                 {formatConnectorName()}
                 <div>
-                  {disableDisconnect && (
+                  {/* {disableDisconnect && (
```

Comment:
why do we comment this code block?

Reply:
we implement this in another feature/ticket. We will add disconnect button for all wallets